### PR TITLE
[node-red] Added serviceAccountName to pod spec

### PR DESCRIPTION
**Pull request by Feliksas**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/node-red/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.0.6-12
 description: Node-RED is low-code programming for event-driven applications
 name: node-red
-version: 3.0.0
+version: 3.0.1
```
Comment (billimek) [inline charts/node-red/Chart.yaml:5]: As we are introducing a feature, can you please bump the version to 3.1.0?

**Review by billimek — COMMENTED** [review]
Just one thing about the version number

File: charts/node-red/Chart.yaml (hunk 1 above)
Comment (Feliksas) [reply charts/node-red/Chart.yaml]: Sure thing

**Review by billimek — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**